nixos/modules/services/networking/nebula.nix

```diff
@@ -196,7 +196,8 @@ in
             before = [ "sshd.service" ];
             wantedBy = [ "multi-user.target" ];
             serviceConfig = {
-              Type = "simple";
+              Type = "notify";
+              NotifyAccess = "main";
```

Comment:
`NotifyAccess = "main";` is implied if `Type = "notify";` is set according to `man systemd.service`.  What about removing this line?

Reply:
Sounds good. Did it because of the Nebula example code but am happy to remove it.